modcc/module.cpp

```diff
@@ -277,24 +277,32 @@ bool Module::semantic() {
     auto initial_api = make_empty_api_method("nrn_init", "initial");
     auto api_init  = initial_api.first;
     auto proc_init = initial_api.second;
+
+    scope_ptr nrn_init_scope = api_init->scope();
```

Comment:
It doesn't look like `nrn_init_scope` is used anywhere. Are you creating it because you need side-effects of the call to `api_init->scope()`?

Reply:
I think it was a mistake. Will remove it.